package.json

```diff
@@ -91,7 +91,15 @@
     "worker-loader": "^2.0.0"
   },
   "resolutions": {
-    "@types/sinon": "5.0.2"
+    "@types/sinon": "5.0.2",
+    "braces": "^2.3.1",
+    "handlebars": "^4.5.3",
+    "js-yaml": "^3.13.1",
+    "lodash": "^4.17.11",
+    "mem": "^4.0.0",
+    "minimatch": "^3.0.2",
+    "serialize-javascript": "^2.1.1",
+    "webpack-dev-server": "^2.9.4"
```

Comment:
Were you unable to resolve this via "dependencies" updates?  That would be the preferred way.  I'd only use "resolutions" as a backup option.

Reply:
Unfortunately it's not possible due to how the dependencies' dependencies are resolved.